### DISCUSSION TITLE
Standup (Thursday, October 20th)

### DISCUSSION THREAD

**Discussion by antonieto** [BBGTC/food-bank-app#15] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on Figma screen flows
 - Read Expo documentation on navigation

**What are your priorities for the next 24 hours?**:
 - Learn more about Expo and contexts

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto for his offer at @Google

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on Figma screen flows
 - Mob programming firstt navigation screens

**What are your priorities for the next 24 hours?**:
 - UI of screens and navigation
 - Record video for Brenda
 - Do homework (algs)

**Blockers**:
 - DSA homeworkd

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished #16 (will open a PR)

**What are your priorities for the next 24 hours?**:
 - Learn about expo apps testing and automated tests
 - Enable branch protection

**Blockers**:
 - None

**Shoutouts**:
 - @CarlosCRG19 for coding like at the speed of light in the morning
